### DISCUSSION TITLE
Configuring Microsoft OAuth applications for a self-hosted installation

### DISCUSSION THREAD

**Discussion by lucascanna** [mia-platform/community#528] [Q&A] [answered]
### What product are you having troubles with?

Console

### What Console version are you using?

v13

### Description

Hello!

I need to set up the OAuth application on my Microsoft Identity Provider for a self-hosted installation. What should the redirect URI be? Should it be the root of my domain or include a specific base path?

Additionally, I need to configure the same for the Console CMS and have the same question regarding the redirect URI.

Thanks!

**Comment by ghost** [comment, marked as answer]
Hi @Lucascanna .

The Redirect URIs for the Application should be:

- for the Console `https://<Console_Host>/oauth/callback`
- for the CMS `https://<CMS_Host>/web-login/oauth/callback`

Hope this helps!
